vault/external_tests/sealmigration/seal_migration_test.go

```diff
@@ -218,6 +481,53 @@ func unsealMigrate(t *testing.T, client *api.Client, keys [][]byte, transitServe
 	}
 }
 
+// awaitMigration waits for migration to finish.
+func awaitMigration(t *testing.T, client *api.Client) {
+
+	timeout := time.Now().Add(60 * time.Second)
```

Comment:
nit: you can use a select with a ticker and timeout. might make it a bit cleaner 👍